docker/gcp_pudl_etl.sh

```diff
@@ -95,17 +95,23 @@ ETL_SUCCESS=${PIPESTATUS[0]}
 
 # if pipeline is successful, distribute + publish datasette
 if [[ $ETL_SUCCESS == 0 ]]; then
-    # Dump outputs to s3 bucket if branch is dev or build was triggered by a tag
-    if [ $GITHUB_ACTION_TRIGGER = "push" ] || [ $GITHUB_REF = "dev" ]; then
-        copy_outputs_to_distribution_bucket
-        ETL_SUCCESS=${PIPESTATUS[0]}
-    fi
-
```

Comment:
I swapped the ordering here since we need the uncompressed SQLite files around to deploy to Datasette. It didn't look like it makes a difference overall, but I could be wrong?

Reply:
This seems fine to me. I guess when we do a datasette deployment we'll run `datasette insspect` on the unzipped data, compress the data to add to the docker image, and then compress the data again to distribute to the buckets. If compressing doesn't take that long this seems fine.

Reply:
Yeah it's not ideal, but 5 minutes in the grand scheme of our 6 hour deployment isn't so bad.